### PR TITLE
Made nginx ingress configmap configurable using values.yaml

### LINKED ISSUE #1337: make nginx configmap content configurable via values
The content of the nginx configmap should be configurable via a variable in the values.yaml

### PR DESCRIPTION
**Pull request by pkavajin**
**What this PR does / why we need it**:
It makes nginx ingress configuration configurable using values.yaml

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #1337 (yay)

**Special notes for your reviewer**:

### CODE REVIEW

File: config/nginx-ingress-controller/values.yaml [hunk 1]
```diff
@@ -13,3 +13,5 @@ nginx:
     image:
       repository: gcr.io/google_containers/defaultbackend
       tag: 1.4
+  config:
+    load-balance: "least_conn"
```
Comment (mrIncompetent) [inline config/nginx-ingress-controller/values.yaml]: Is this a sane default?

Comment (pkavajin) [reply config/nginx-ingress-controller/values.yaml]: Default for usual nginx ingress and actuall most lbs

Comment (mrIncompetent) [reply config/nginx-ingress-controller/values.yaml]: cc @alvaroaleman

Comment (alvaroaleman) [reply config/nginx-ingress-controller/values.yaml]: Doesn't really make a difference as the kube-proxy will distribute it round-robin. Do we have a reason to expose this?

Comment (mrIncompetent) [reply config/nginx-ingress-controller/values.yaml]: Please just comment it out. So a user knows how to use it

**Review by mrIncompetent — APPROVED** [review]

**Pull request merged**